### PR TITLE
ios (iPhone6) and iPhone XS tiles_scroll_perf tests

### PR DESCRIPTION
**Pull request by liyuqian**
For https://github.com/flutter/flutter/issues/31086.

Also test if our mac8 with iPhone XS is working
(https://github.com/flutter/flutter/issues/34590).

### CODE REVIEW

File: dev/devicelab/manifest.yaml [hunk 1]
```diff
@@ -146,6 +146,20 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  tiles_scroll_perf_ios__timeline_summary:
+    description: >
+      Measures the runtime performance of the tiles tab in the Complex Layout sample app on Android.
+    stage: devicelab
+    flaky: true
+    required_agent_capabilities: ["mac/ios"]
```
Comment (digiter) [inline dev/devicelab/manifest.yaml]: What's the difference between ios and iphonex? It's confusing to see both of them.

File: dev/devicelab/manifest.yaml [hunk 2]
```diff
@@ -146,6 +146,20 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  tiles_scroll_perf_ios__timeline_summary:
+    description: >
+      Measures the runtime performance of the tiles tab in the Complex Layout sample app on Android.
```
Comment (digiter) [inline dev/devicelab/manifest.yaml]: This is not on Android, also applies to below comment.

File: dev/devicelab/manifest.yaml (hunk 1 above)
Comment (liyuqian) [reply dev/devicelab/manifest.yaml]: "mac/ios" is iPhone 6, "mac/ios32" is iPhone 4s.

I'm adding the device model to all ios task descriptions.

File: dev/devicelab/manifest.yaml (hunk 2 above)
Comment (liyuqian) [reply dev/devicelab/manifest.yaml]: Done.

**Review by digiter — APPROVED** [review]

**Comment by digiter** [comment]
LGTM

File: dev/devicelab/bin/tasks/tiles_scroll_perf_ios__timeline_summary.dart [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+import 'package:flutter_devicelab/tasks/perf_tests.dart';
+import 'package:flutter_devicelab/framework/adb.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+
+Future<void> main() async {
+  deviceOperatingSystem = DeviceOperatingSystem.ios;
+  await task(createTilesScrollPerfTest());
+}
+
```
Comment (goderbauer) [inline dev/devicelab/bin/tasks/tiles_scroll_perf_ios__timeline_summary.dart]: nit: remove the extra blank line?

File: dev/devicelab/bin/tasks/tiles_scroll_perf_iphonex__timeline_summary.dart [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+import 'package:flutter_devicelab/tasks/perf_tests.dart';
+import 'package:flutter_devicelab/framework/adb.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+
+Future<void> main() async {
+  deviceOperatingSystem = DeviceOperatingSystem.ios;
+  await task(createTilesScrollPerfTest());
+}
+
```
Comment (goderbauer) [inline dev/devicelab/bin/tasks/tiles_scroll_perf_iphonex__timeline_summary.dart]: nit: remove the extra blank line?

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**